Makefile

```diff
@@ -27,32 +26,9 @@ cover: $(COVER_FILE) ## Output coverage in human readable form in html
 	go tool cover -html=$(COVER_FILE)
 	rm -f $(COVER_FILE)
 
-
 .PHONY: lint
 lint: tools ## Check the project with lint
-	staticcheck ./...
-
-.PHONY: vet
-vet: ## Check the project with vet
-	go vet ./...
-
-.PHONY: fmt
-fmt: ## Run go fmt for the whole project
-	test -z $$(for d in $$(go list -f {{.Dir}} ./...); do gofmt -e -l -w $$d/*.go; done)
-
-.PHONY: imports
-imports: $(GOIMPORTS) ## Check and fix import section by import rules
-	test -z $$(for d in $$(go list -f {{.Dir}} ./...); do goimports -e -l -local $$(go list) -w $$d/*.go; done)
-
-.PHONY: cyclomatic
-cyclomatic: tools ## Check the project with gocyclo for cyclomatic complexity
-	gocyclo -over 10 `find . -type f -iname '*.go' ! -iname "*_test.go" -not -path '*/\.*'`
-
-.PHONY: static_check
-static_check: fmt imports vet lint cyclomatic ## Run static checks (fmt, lint, imports, vet, ...) all over the project
-
-.PHONY: check
```

Comment:
Could you, please, explain, why this target was removed? It could run tests with other static checks.

Reply:
All this checks are included into golangci-lint and will be run in the `lint` step.

Reply:
Nope, unfortunately there was also test target, so could you please return it to have ability to run all quality checks in one step?

I mean something like:
```
.PHONY: check
check: lint test ## Check project with linters and unit tests`